### PR TITLE
Increased indent in Interactable profiles, the Theme label under Targ…

### PR DESCRIPTION
**Pull request by killerantz**
Increased indent in Interactable profiles, the Theme label under Target was not consistent when Dimensions were increased, like check boxes.

Issue #3997 

# Overview
https://user-images.githubusercontent.com/22085553/56380628-8ccad500-61c7-11e9-9c84-caef2582874a.png

Shows the first Theme label under target is not indented, but the second one is. They should be consistent and sense they are tied to Target, they should all be indented.

# Changes
Increase intent and added an indent reset at the top of the loop.

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs [hunk 1]
```diff
@@ -318,6 +318,7 @@ public virtual void RenderCustomInspector()
                     for (int t = 0; t < themes.arraySize; t++)
                     {
                         SerializedProperty themeItem = themes.GetArrayElementAtIndex(t);
+                        EditorGUI.indentLevel = indentOnSectionStart + 2;
                         EditorGUILayout.PropertyField(themeItem, new GUIContent("Theme", "Theme properties for interaction feedback"));
 
                         // TODO: we need the theme and target in order to figure out what properties to expose in the list
```
Comment (julenka) [inline Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs]: Are these TODO items still needed?

File: Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs [hunk 2]
```diff
@@ -338,11 +339,11 @@ public virtual void RenderCustomInspector()
 
                             SerializedProperty hadDefault = sItem.FindPropertyRelative("HadDefaultTheme");
                             hadDefault.boolValue = true;
-                            EditorGUI.indentLevel = indentOnSectionStart + 2;
+                            EditorGUI.indentLevel = indentOnSectionStart + 3;
 
                             string prefKey = themeItem.objectReferenceValue.name + "Profiles" + i + "_Theme" + t + "_Edit";
                             bool showSettings = EditorPrefs.GetBool(prefKey);
-
+
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs]:
```suggestion

```

**Comment by killerantz** [comment]
I’ll target the correct branch, thanks for catching that. I’ll also review the TODOs. I think most of them can go now.

File: Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs (hunk 1 above)
Comment (killerantz) [reply Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs]: Removed the TODO notes.

**Comment by keveleigh** [comment]
Closing and reopening to hopefully trigger the license status check.

**Pull request merged**